templates/includes/components/grid_boards.html

```diff
@@ -0,0 +1,71 @@
+<input type="search" id="searchBoards" placeholder="Search boards" oninput="filterBoardsList()">
+<ul id="boardsGrid" class="grid-list u-clearfix col-12 u-no-padding--left u-no-margin--left">
+    {% for page in pages %}
+        <li class="col-6 grid-list__item{% if forloop.last %} last-item{% endif %}">
+          <h3>{{ page.title }}</h3>
+          <!-- Do not display tags on grid items -->
+          <p class="board-tags" style="display:none;">{{ page.tags }}</p>
+          <div class="grid-list__description">
+            <ul class="p-list--divided">
+              {% for d in page.distributions %}
+                <li class="p-list__item  is-ticked">
+                    <!--
```

Comment:
Can these comments be changed to a Django comment block, they currently get printed out on the page many times.
```
{% comment %}
comments here
{% endcomment %}
```

Reply:
Hah good point, I forgot about the "comment" tag. Changing.